src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -3154,24 +3154,24 @@ object Desugar {
       maybeRegister(s, context)
 
       // n has occur first in order that function inverse properly works
-      s"${n}_${context.pkgInfo.viperId}_$postfix${scopeMap(s)}" // deterministic
+      s"${n}_$postfix${scopeMap(s)}" // deterministic
```

Comment:
As far as I can tell, the `scopeMap` returns a unique identifier only within a package. I.e. it is not unique in the resulting Viper encoding.
This should however be fine as long as the resulting name is only used in a non-global scope. It's currently used for variables, in and out parameters, and receivers, which should be fine. Nevertheless, I would add a short documentation to this function

Reply:
done :)